README.md

```diff
@@ -1,13 +1,13 @@
-# Data structures 
+# Data structures
```

Comment:
In place of "Data structures" add "Data Structures and Algorithms"

Reply:
The action only edits a portion of the readme, so you can change the rest of the readme easily.